frontend/src/components/Footer/HallOfFame.tsx

```diff
@@ -0,0 +1,94 @@
+import { Dialog, DialogTitle, Typography, IconButton, DialogContent, makeStyles, Grid, Box } from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+
+const rubberdokMembers: {
+  [year: string]: {
+    name: string;
+    class: number;
+    position?: string;
+  }[];
```

Comment:
Two things:
 - Move this to `constants.ts`
 - Make a type
```typescript
type Member = {
  name: string;
  class: number;
  position?: string;
}
```
 - And then
```typescript
rubberdokMembers: Record<string, Member>[]
```

Reply:
Added in 1085bf9.